src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -89,7 +95,7 @@ export default class GroupComparisonStore {
         }
     }
 
-    readonly unfilteredGroups = remoteData<ComparisonGroup[]>({
+    readonly _remoteGroups = remoteData<ComparisonGroup[]>({
```

Comment:
why the underscore?

Reply:
just to encourage that its not to be used, its just part of the computation of the groups. what should be used is availableGroups and activeGroups. let me know if you have a better way to do this, I couldn't figure one out.